src/client/common/experiments/service.ts

```diff
@@ -87,6 +87,14 @@ export class ExperimentService implements IExperimentService {
             return false;
         }
 
+        // If user is already in Native Notebook experiment, then they cannot be in Custom Editor experiment.
```

Comment:
I swear we had this somewhere else already? Maybe it was in the wrong spot before

Reply:
extensionActivation.ts?

Reply:
That location just gives a warning if you are in both and asks you to disable one. I think the intention was that we only though that could happen if you opted in manually.

Reply:
Do we know that this issue was due to the user opting in? This change doesn't look like it guards against control tower issues.

Reply:
> This change doesn't look like it guards against control tower issues.

This will ensure we don't even bother checking.
Hence telemetry will not be sent for this user for this experiment.

Reply:
Ahh, I didn't see where you had added the Insiders => NativeNotebook forcing function. Yeah, this works since that force is in the optinoptout function.